src/Microsoft.Azure.EventHubs/EventDataSender.cs

```diff
@@ -40,6 +44,50 @@ internal static int ValidateEvents(IEnumerable<EventData> eventDatas)
             return count;
         }
 
+        async Task<EventData> ProcessEvent(EventData @event)
```

Comment:
>@event [](start = 53, length = 6)

Let's use eventData here.

Reply:
Perfect will check if in other place there is a @event also

Reply:
Also changed here https://github.com/Azure/azure-event-hubs-dotnet/pull/324/files#diff-a221f37e8c378f551b70067d20e88d6eR82